okhttp/src/main/kotlin/okhttp3/Cache.kt

```diff
@@ -605,7 +605,7 @@ class Cache internal constructor(
         for (i in 0 until length) {
           val line = source.readUtf8LineStrict()
           val bytes = Buffer()
-          bytes.write(line.decodeBase64()!!)
+          line.decodeBase64()?.let { bytes.write(it) } ?: throw java.lang.NullPointerException("The certificate is not base64 encoded -- $line")
```

Comment:
Why? and which string `line`?

Reply:
I checked the code of `decodeBase64` and it seems to fall back to `null` if the input is corrupt. So I think the current way @brunoescalona put it so far should be fine. And our issue is that it crashes here because of the thing being null, so I think no need to check for empty string (at least for now)